Services/ProductsService.js

```diff
@@ -1,14 +1,31 @@
 const getAllProducts = async (connection) => {
     return await connection.query("SELECT `name`, `price`, `SKU` FROM `products`;")
-
 }
 
 const addSingleProduct = async(connection, sku, name, price, stock_level) => {
     return await connection.query('INSERT `products` (`SKU`, `name`, `price`,`stock_level`) VALUES' + `('${sku}', '${name}', '${price}', '${stock_level}');`)
 }
 
+const getSingleProduct = async (connection, SKU) => {
+    const verifiedSKU = async (SKU) => {
```

Comment:
I think we currently have duplicate sanitiseSKU functions - one in this file and one in the sanitiseValidateService. I think we need to keep the sanitise validate logic in the SanitiseValidateService and then call it in the Controller, and keep the Product Service file for the connection.querys only

Reply:
Agreed, Amended